cfme/base/credential.py

```diff
@@ -8,17 +8,47 @@
 
 
 class FromConfigMixin(object):
-    @classmethod
-    def from_config(cls, key):
-        creds = deepcopy(conf.credentials[key])
+    @staticmethod
+    def rename_properties(creds):
```

Comment:
unless we plan to expose we should make this one a global function perhaps with a `_` prefix